MergerLogic/Utils/ImageUtils.cs

```diff
@@ -8,7 +8,7 @@ public static bool IsTransparent(MagickImage image)
         {
             using var pixels = image.GetPixels();
             // Check pixels to see if all are transparent
-            return pixels.Select(pixel => pixel.ToColor()).Any(color => color?.A != 255);
+            return image.HasAlpha && pixels.Select(pixel => pixel.ToColor()).Any(color => color?.A != 255);
```

Comment:
why do you get the pixels when image has no alpha? 😢

Reply:
Changed to be a guard clause.